docs/modules/user-guide/nav.adoc

```diff
@@ -1,4 +1,5 @@
 * xref:index.adoc[]
+* xref:devfile-properties.adoc[]
```

Comment:
How does the placement of this new doc look in the TOC?

Reply:
The placement is ok but I wonder if the title should be something like "Devfile quick reference" (trying to differentiate this between the spec/schema link and the stack author's guide.